src/Components/test/E2ETest/package.json

```diff
@@ -11,11 +11,9 @@
   },
   "author": "",
   "license": "MIT",
-  "dependencies": {
-    "sauce-connect-launcher": "^1.3.1",
-    "selenium-standalone": "^7.1.0"
-  },
   "devDependencies": {
+    "sauce-connect-launcher": "^1.3.1",
+    "selenium-standalone": "^9.1.1",
```

Comment:
Does this mean https://github.com/dotnet/aspnetcore-internal/blob/main/docs/operations/selenium_update.md should be updated?